util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
In default, %OS% is set as Windows_NT. AFAIK, I don't know the toolchain or make that set %OS% as mingw32.

![](https://i.gyazo.com/416911245686aaaee84ef5ed1f0b5305.png)